src/main/java/org/localhost/pizzeria/supplies/system/scheduler/SupplyCheckScheduler.java

```diff
@@ -11,29 +14,40 @@
 
 @Service
 @EnableScheduling
+@Slf4j
 public class SupplyCheckScheduler {
     private final SupplySystemQueryService supplySystemQueryService;
-    private static final PizzeriaLogger log = PizzeriaLogger.getInstance();
+    private final RabbitTemplate rabbitTemplate;
 
 
-    public SupplyCheckScheduler(SupplySystemQueryService supplySystemQueryService) {
+    @Value("${rabbitmq.exchange.supply-check}")
+    private String supplyCheckExchange;
+
+    @Value("${rabbitmq.routing-key.supply-check}")
+    private String supplyCheckRoutingKey;
+
+
+    public SupplyCheckScheduler(SupplySystemQueryService supplySystemQueryService, RabbitTemplate rabbitTemplate) {
         this.supplySystemQueryService = supplySystemQueryService;
+        this.rabbitTemplate = rabbitTemplate;
     }
 
     @Scheduled(fixedDelayString = "${scheduler.supply-check.interval}")
     public void checkSuppliesStatus() {
```

Comment:
wedlug mnie tutaj nie powinien byc interwal tylko cron 
dlaczego?
czy napewno potrzebujesz sprawdzac stan magazynowy co dajmy na to nawet pare godzin? raczej wedlug biznesu powinno to byc tak zrobione ze albo masz cos co pozwala ci na wytworzenie zamowien na caly dzien albo nie. Mozesz to sprawdzic raz dziennie wedlug jakiegos limitu nie potrzebujesz generowac ruchu co x